server/app.js

```diff
@@ -406,6 +407,7 @@ if(isLocalEnvironment){
 //Render the page
 const templateFn = require('./../client/template.js');
 app.use(asyncHandler(async (req, res, next)=>{
+	const brew = _.defaults(req.brew, DEFAULT_BREW);

```

Comment:
@G-Ambatte Do we still need this here? Is this just for the case we haven't `get`'d a brew yet? I.e., for new brews?